bindings/python/CONTRIBUTING.md

```diff
@@ -55,15 +55,15 @@ The following shows minimal example of relation between C++ library source files
 files. It shows link between (1) Cython binding files (`nfa.pxd`, `nfa.pyx`) and C++ library files 
 (`nfa-plumbing.h`), (2) Cython binding class (`Nfa`) and C++ library class (`mata::nfa::Nfa`; 
 in binding refered as `CNfa`), and (3) Cython binding function (`union` that takes Python 
-classes `Nfa`) and C++ library function (`uni` that takes C++ classes `mata::nfa::Nfa`, 
+classes `Nfa`) and C++ library function (`unite` that takes C++ classes `mata::nfa::Nfa`, 
 in binding refered as `c_uni` and `CNfa` respectively)
 
 ```c++
 // @file: nfa-plumbing.h
 
 // Minimal specification of C++ function, we want to wrap in Binding
 namespace mata::nfa::plumbing {
-    inline void uni(Nfa *unionAutomaton, const Nfa &lhs, const Nfa &rhs) { *unionAutomaton = uni(lhs, rhs); }
+    inline void uni(Nfa *unionAutomaton, const Nfa &lhs, const Nfa &rhs) { *unionAutomaton = unite(lhs, rhs); }
```

Comment:
Was this on purpose? I don't remember seeing renaming of `uni` function anywhere? Is this mistake of auto-rename in Clion?

Reply:
no, sorry. I made unite in c++ for a three parameter version, I did not want to change this one.

Reply:
What about all the other occurrences of `uni` that were renamed to `unite`. Was that also unintentional? I think we wanted to rename all occurrences of `uni` to something better. The last time we talked about that, we mentioned `unite`, no?

Reply:
I reverted the ones in strange places. 
The renaming is a hard problem. 
unite does not return stuff, so I like to call it unite. 
I am still for renaming the uni that returns to onion, but I guess I will not win this now, so I would keept it to another PR.